File: Wirtualny Swiat C++/animals/Wolf.cpp
//
// Created by Dorota on 17.04.2025.
//

#include "Wolf.h"

#include "../GameWorld.h"

Wolf::Wolf(int x, int y, GameWorld* game_world): Animal(9, 5, x, y, game_world, 'W') {

}


void Wolf::draw() {
    cout << symbol;
}

void Wolf::action() {
    Animal::action(); //dziedziczenie ruchu
}

File: Wirtualny swiat-java/Animals/Wolf.java
package Animals;

import World.Organism;
import World.Animal;
import World.GameWorld;
import World.Point;

public class Wolf extends Animal {
    public Wolf(int x, int y, GameWorld gameWorld) {
        super(9, 5, x, y, gameWorld, 'W');
    }

    @Override
    public void draw() {
        System.out.print(symbol);
    }

    @Override
    public void action() {
        super.action();
    }
}
